components/more-less/more-less.js

```diff
@@ -20,12 +20,6 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	static get properties() {
 		return {
-			/**
-			 * The gradient color of the blurring effect
-			 * @type {string}
-			 */
-			blurColor: { type: String, attribute: 'blur-color' },
```

Comment:
Was the purpose of this to allow consumers to match the blur to non-white background colors, and `mask-image` essentially handles that for us?

Reply:
Correct